### PR TITLE
Changes on F2F minutes

### PR DESCRIPTION
**Pull request by iherman**
- turned them into approved
- make the changes proposed in https://www.w3.org/mid/1529A6E2-2717-4D1E-B15D-D81803278BBB@greggkellogg.net

### CODE REVIEW

**Comment by davidlehn** [comment]
These emails and prs are getting confusing!  I'm pretty sure you all mean `api#26` vs `syntax#26`?

The syntax one is about examples and closed months ago: https://github.com/w3c/json-ld-syntax/issues/26. API one is about blank nodes: https://github.com/w3c/json-ld-api/issues/26.

Remember the good ol days when the issues were all in one repo? ;-)

**Comment by gkellogg** [comment]
Yes, indeed, my mistake. It should be w3c/json-ld-api#26.

File: _minutes/2019-02-08-json-ld.md [hunk 1]
```diff
@@ -1038,7 +1036,7 @@ See also the [Agenda](https://tinyurl.com/ycsvtmu9) and the [IRC Log](https://ww
 
 **Gregg Kellogg:** fragment identifiers would be a better solution.  
 
-> **Proposed resolution: close framing#27 wontfix, as there's no justification for the required RDF layer requirement that the blank node identity of the named graph is the default subject of the triples in the graph** *(Rob Sanderson)*
+> **Proposed resolution: close syntax#26 wontfix, as there's no justification for the required RDF layer requirement that the blank node identity of the named graph is the default subject of the triples in the graph** *(Rob Sanderson)*
```
Comment (gkellogg) [inline _minutes/2019-02-08-json-ld.md]: Should be api#26.

File: _minutes/2019-02-08-json-ld.md [hunk 2]
```diff
@@ -1057,7 +1055,7 @@ See also the [Agenda](https://tinyurl.com/ycsvtmu9) and the [IRC Log](https://ww
 
 > *Adam Soroka:* +0
 
-> ***Resolution #15: close framing#27 wontfix, as there's no justification for the required RDF layer requirement that the blank node identity of the named graph is the default subject of the triples in the graph*** {: #resolution15 .resolution}
+> ***Resolution #15: close syntax#26 wontfix, as there's no justification for the required RDF layer requirement that the blank node identity of the named graph is the default subject of the triples in the graph*** {: #resolution15 .resolution}
```
Comment (gkellogg) [inline _minutes/2019-02-08-json-ld.md]: Should be api#26.

File: _minutes/2019-02-08-json-ld.md [hunk 3]
```diff
@@ -1079,7 +1077,7 @@ See also the [Agenda](https://tinyurl.com/ycsvtmu9) and the [IRC Log](https://ww
 * [Resolution #12](#resolution12): Use `@type:@none` to force compaction to generate object values ; Ivan and Adam will review for understandability
 * [Resolution #13](#resolution13): Close framing #31, wontfix, with request for further discussion if they can provide more information to support its inclusion
 * [Resolution #14](#resolution14): Verify that `@nest` solution for #19 solves issue in #128 and defer until then
-* [Resolution #15](#resolution15): close framing#27 wontfix, as there's no justification for the required RDF layer requirement that the blank node identity of the named graph is the default subject of the triples in the graph
+* [Resolution #15](#resolution15): close syntax#26 wontfix, as there's no justification for the required RDF layer requirement that the blank node identity of the named graph is the default subject of the triples in the graph
```
Comment (gkellogg) [inline _minutes/2019-02-08-json-ld.md]: Should be api#26.

**Comment by iherman** [comment]
This issue was discussed in [a meeting](https://www.w3.org/2018/json-ld-wg/Meetings/Minutes/2019/2019-02-15-json-ld#section1).

- `RESOLVED:  Approve the minutes of the F2F, with the changes for the framing issue *actually discussed rather than the issue number recorded* {: #resolution1 .resolution}`
<details><summary><i class="differentiate">View the transcript</i></summary>
Approve minutes of Face to Face<br/>
Rob Sanderson: there are some questions here<br/>
… on Friday, we discussed some complicated issue about framing,<br/>
… but the issue referenced in the minutes seems to be the wrong one<br/>
Ivan Herman: I put a PR with the change fixing this: <a href="https://github.com/w3c/json-ld-wg/pull/28">https://github.com/w3c/json-ld-wg/pull/28</a><br/>
Gregg Kellogg: the issue that we originally discussed was the overloading of bnode<br/>
… while the issue we referenced is the use of bnode in frames, which I don’t think is controversial<br/>
Proposed resolution: Approve the minutes of the F2F, with the changes for the framing issue actually discussed rather than the issue number recorded (Rob Sanderson)<br/>
Dave Longley: +1<br/>
Pierre-Antoine Champin: +1<br/>
Rob Sanderson: +1<br/>
Harold Solbrig: +1<br/>
Simon Steyskal: +1<br/>
Ivan Herman: +1<br/>
Benjamin Young: +1<br/>
Gregg Kellogg: +1<br/>
David I. Lehn: +1<br/>
David Newbury: +1<br/>
Adam Soroka: +1<br/>
Resolution #1: Approve the minutes of the F2F, with the changes for the framing issue *actually discussed rather than the issue number recorded* {: #resolution1 .resolution}<br/>
</details>

**Pull request merged**